### PR TITLE
feat(taskmanager): add filter and sort

### PR DESCRIPTION
**Pull request by OverIoad**
Sort task #52

### CODE REVIEW

**Comment by OverIoad** [comment]
There is still a bug in the filtering process. Although the sorting algorithm seems to be working as expected, it hasn't been fully tested yet.

File: src/components/elements/taskmanager.tsx [hunk 1]
```diff
@@ -224,6 +236,106 @@ export const TaskManager = ({ project_id }: TaskManageMentOverviewProp) => {
...
+      try {
+        const response = await fetch(url, options);
+        const data = await response.json();
+
+        // Map the data to extract the task ids as a string array
+        const taskIds = data.map((task: { id: string }) => task.id);
+
+        console.log('taskIds: ', taskIds);
+
+        // Return the task ids array
```
Comment (bunnybunbun37204) [inline src/components/elements/taskmanager.tsx:276]: this func didnt work on my machine

**Review by bunnybunbun37204 — CHANGES_REQUESTED** [review]
this func didnt work on my pc, or i missed something ?

**Review by bunnybunbun37204 — APPROVED** [review]
Approve

**Pull request merged**